api/src/main/resources/client_api.yaml

```diff
@@ -168,13 +168,22 @@ definitions:
     required:
       - bigqueryProject
       - bigqueryDataset
+      - columns
 
     properties:
       sql:
         description: >
           Google SQL to use when querying the CDR. If empty, it means no participants can possibly
           match the data table specification, and an empty data table should be returned.
         type: string
+      columns:
+        description: >
+          An array of names to be used for the columns being returned by the query.
```

Comment:
How is this different from the values in the select query? Presumably gbq uses the SELECT column names by default for its column labels?

Is it possible to instead implement this with aliases in the SELECT statement? I worry about customizing the logic too far away from standard gbq usage.

Reply:
Two things:

1) BigQuery doesn't support using dots (e.g. "foo.bar") in column aliases... you can only use letters, numbers, and underscores. That's much uglier / harder to read / doesn't match our way of constructing column names in the client library... so we'd prefer to give dot notation names in the final results. This will allow us to do that.

2) In some cases we don't actually have any matching participants and don't run any SQL (e.g. cohort status = INCLUDED but you didn't mark any participants as INCLUDED in a cohort review.) We still want to return a data frame with the correct columns in that context -- see #workbench-beta about this.

So that's why I'm adding these.